package.json

```diff
@@ -31,5 +31,17 @@
         "eslint-config-airbnb-base": "^12.1.0",
         "eslint-plugin-import": "^2.9.0",
         "jest": "^22.4.2"
+    },
+    "resolutions": {
+        "mem": "^5.1.1",
+        "lodash": "^4.17.15",
+        "stringstream": "^1.0.0",
+        "js-yaml": "^3.13.1",
+        "just-extend": "^4.0.2",
+        "extend": "^2.0.2",
+        "merge": "^1.2.1",
+        "randomatic": "^3.1.1",
+        "cryptiles": "^4.1.3",
```

Comment:
I think this might have been fixed already by @KensoDev, but in any case can we fix this by bumping Jest instead?

Chain: `jest-cli` -> `jest-environment-jsdom` -> `jsdom` -> `request` -> `hawk` -> `cryptiles`